libs/ledger-live-common/src/platform/providers/RampCatalogProvider/index.tsx

```diff
@@ -56,7 +54,7 @@ export function RampCatalogProvider({
     }));
 
     try {
-      const value = await api.fetchRampCatalog(provider);
+      const value = await api.fetchRampCatalog("production");
```

Comment:
This value should not be hardcoded, the user should still be able to chosse between `production` and `staging` ramp catalog through the UI.
FYI: the ramp catalog is different from the manifest (live-app) one. This one is still hosted on S3 (cf. [here](https://github.com/LedgerHQ/ledger-live-assets/tree/develop/platform/trade)) and is not planned to be moved soone (cf. [here](https://ledgerhq.atlassian.net/wiki/spaces/PTX/pages/3727261997/ARCH+BUY+-+Architecture+information#With-backend-support-proposal-(Long-term)))

One solution would be to revert (part of the) changes made to the settings to retrofit the `setCatalogProvider` option to be used only for the ramp catalog now.